src/main/java/games/strategy/engine/framework/GameDataFileUtils.java

```diff
@@ -9,21 +9,22 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
-import games.strategy.engine.ClientContext;
+import games.strategy.triplea.settings.ClientSetting;
 
 /**
  * A collection of utilities for working with game data files.
  */
 public final class GameDataFileUtils {
   private static final IOCase DEFAULT_IO_CASE = IOCase.SYSTEM;
 
-  private static final SaveGameFormat DEFAULT_SAVE_GAME_FORMAT =
-      ClientContext.gameEnginePropertyReader().useNewSaveGameFormat()
-          ? SaveGameFormat.NEW
-          : SaveGameFormat.CURRENT;
-
   private GameDataFileUtils() {}
 
+  private static SaveGameFormat getDefaultSaveGameFormat() {
+    return ClientSetting.TEST_USE_NEW_SAVE_GAME_FORMAT.booleanValue()
+        ? SaveGameFormat.NEW
```

Comment:
This was existing, though less context dependent names may be better compared to 'new' and 'current' 
As a possible suggestion for renaming: 'current' -> 'serialization'; 'new' -> 'proxy_serialization'